projects/demo/src/modules/components/editor/custom-tool/color-picker/editor-color-picker.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiDocExample} from '@taiga-ui/addon-doc';
+import {defaultEditorExtensions, TUI_EDITOR_EXTENSIONS} from '@taiga-ui/addon-editor';
+
+@Component({
+    selector: 'editor-color-picker-tool',
+    templateUrl: './editor-color-picker.component.html',
+    encapsulation,
+    changeDetection,
+    providers: [
+        {
+            provide: TUI_EDITOR_EXTENSIONS,
```

Comment:
You also provide it inside `TuiEditorColorPickerToolExample1`.

Should we drop it here ?

Reply:
fixed